### ISSUE TITLE
[Bug?] Steam Game Pad not showing up in Big Picture, yet kinda works? - fresh upgrade 19.04

### ISSUE THREAD

**Issue by BloodyIron** [ValveSoftware/steam-for-linux#6225]
I'm getting pretty fed-up with the lack of polish and testing on the out of box experience for Steam Big Picture gaming and gamepads.

I'm on a new rig I built a few months ago with Ubuntu 18.10, and just today upgraded to 19.04.

I haven't used my Steam Game Pad on my new rig, so no drivers installed.

I go into Big Picture to take advantage of the Overcooked 2 discount, and well, the gamepad "just works" in the Big Picture menus, but when I go into the game, it doesn't. I can move around, but the buttons don't register.

Okay, go back to the gamepad section oof Big Picture, and it doesn't even see my device.

Guys, this is fucking 2019. This out of the box experience is unacceptable. I've had to put up with inconsistent performance in Big Picture for years now, and I think it's about time you guys get this sorted in the main repos already. You got STEAM in the main Ubuntu repos, now make the gamepad stuff required for that to be installed, and make all the gamepad support on by default.

Stop fucking around. You wanted to make this a streamlined experience whereby a computer could enter the couch space, and you have a lot done, but the UX polish is extremely lacking in areas that are _immediately_ noticeable.

I love you guys at VALVe, but you need to stop dropping the ball on UX polish here. How the fuck have you guys still not figured out how to make this awesome out of the box with 19.04 or even 18.10? I shouldn't have to scour tJhe internet for something that's so well built in other regards. (oh and btw, why did your design not include a place to stow the USB dongle exactly?).

I'm willing to help you help me here, but I'm getting so exhausted. Trying to sit down to play a nice game on Big Picture with my very well thought-out gamepad, only to find time and time again that I have to figure some new bullshit out.

To put it another way. This would never happen with a PS4, Xbox One or Nintendo Switch. And you guys should hold yourselves to a higher standard than that. Because I know you do. So act like it.

What do we do to get this out of the box experience to "just work" already? How can I help?

**Comment by kisak-valve** [comment]
Hello @BloodyIron, are you using the Valve-provided steam package or the distro-provided steam package? If you're using the distro-provided steam package, do you also have the `steam-devices` package installed?

**Comment by BloodyIron** [comment]
When I first installed 18.10, went to CLI, apt install steam, so from the Ubuntu main repos.

I do not have steam-devices installed, if that's drivers for controllers, why not make that a dependence for steam?

Should I install that?

I just really want this to be a really awesome experience for anyone.

**Comment by kisak-valve** [comment]
`steam-devices` includes udev rules for various hardware, and yes, it should be installed if you're using a Steam controller and the distro-provided Steam package. The dependency question should be directed at your distro package mantainer(s) for Steam.

**Comment by BloodyIron** [comment]
@kisak-valve I have not found an effective way to get anyone's attention at Canonical, I can get your guys' attention way easier than theirs, so I don't even know where to begin to convince them to make that change. Any way you can pass it through the grapevine please? :)

I'll try that package now. Do you know if that package also impacts Bluetooth PS3 controllers? I tried one I have recently too, and it did some wonky sensor stuff, so I'm hoping this package may improve that (unsure).

**Comment by BloodyIron** [comment]
Okay so installed steam-devices package and the controller still doesn't show up in big picture mode. Even rebooted computer, no change. Now the controller doesn't even interact with the main menu at all (previously it did)

**Comment by kisak-valve** [comment]
Hello @BloodyIron, are you still experiencing this issue on an up to date system?

**Comment by BloodyIron** [comment]
Hey @kisak-valve I haven't observed recent issues with my Steam Gamepad, so I'm okay with considering this resolved. Unsure what did the trick, but I have used the gamepad in the last year or so on my Steam Link, and it seemed to work just fine. The system it connected to recently is Ubuntu 20.04 with regular updates, same one as before, so unsure if that still relies on the package you pointed me to, or not. I have not wiped that computer with a fresh install in multiple years, so it's unclear. Hope that helps!

Thanks for checking in btw, you guys at VALVe are great! <3 So excited for the Steam Deck launch and EAC/BattleEye support! :DDDDDDDDDDDD

**Comment by kisak-valve** [comment]
Glad to hear it's working for you now. Closing.
